pages/en/guides/diagnostics/poor-performance/index.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
why it's added? it's already migrate take a look to `en/learn/diagnostics/poor-performance`